### PR TITLE
Add a keybinding to break straight into the debugger

### PR DESCRIPTION
**Pull request by zadjii-msft**
Have you ever wanted to debug the Terminal, but weren't sure which of
your Terminal windows was the one you needed to attach to? Now you don't
need to worry! Simply execute the `breakIntoDebugger` action, and the
Terminal will `DebugBreak()` for you!

## PR Checklist
* [x] Scratches a silly itch
* [x] I work here
* [x] Lol there aren't tests for this
* [x] We could always just _not_ document this, right?

## Detailed Description of the Pull Request / Additional comments

This requires that the user has set `"debugFeatures": true`

## Validation Steps Performed
Added a command
```json
{
    "command": "breakIntoDebugger",
    "keys": "ctrl+alt+shift+f1",
    "name": "DebugBreak()"
},
```
And verified that thing pops open the post-mortem debugger (windbg).

### CODE REVIEW

**Comment by DHowett** [comment]
HAHAHAHA

This is so clever I could die. I love it.

File: src/cascadia/TerminalApp/AppActionHandlers.cpp [hunk 1]
```diff
@@ -550,4 +550,15 @@ namespace winrt::TerminalApp::implementation
             actionArgs.Handled(true);
         }
     }
+
+    void TerminalPage::_HandleBreakIntoDebugger(const IInspectable& /*sender*/,
+                                                const ActionEventArgs& actionArgs)
+    {
+        if (_settings.GlobalSettings().DebugFeaturesEnabled())
```
Comment (DHowett) [inline src/cascadia/TerminalApp/AppActionHandlers.cpp:557]: YES! THE SECOND DEBUG FEATURE!

File: src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw [hunk 2]
```diff
@@ -345,4 +345,7 @@
   <data name="ToggleRetroEffectCommandKey" xml:space="preserve">
     <value>Toggle retro terminal effect</value>
   </data>
+  <data name="BreakIntoDebuggerCommandKey" xml:space="preserve">
+    <value>Break into the debugger</value>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw:349]: feel free to localization lock this entire phrase -- no need to have them deal w/ it, yeah?

**Review by DHowett — APPROVED** [review]

**Review by miniksa — APPROVED** [review]
Yeah I mean ok.

File: src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw (hunk 2 above)
Comment (zadjii-msft) [reply src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw:349]: sure, I'll bump it

Comment (DHowett) [reply src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw:349]: i decided it was ok to ship like this

**Pull request merged**